### PR TITLE
nats: Add a test for memory hotplug

### PR DESCRIPTION
**Pull request by rbradford**
Test that memory hotplug works by seeing if the MemTotal in the guest
increases after inserting a DIMM.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Comment by rbradford** [comment]
Memory only for now..CPU hotplug is consistently failing on the Jenkins VM so need to root-cause that separatelly.

File: tools/CI/nats/nemu_test.go [hunk 1]
```diff
@@ -398,3 +399,49 @@ func TestQMPQuit(t *testing.T) {
...
+		ctx, cancelFunc := context.WithTimeout(context.Background(), 120*time.Second)
+		err := q.startQemu(ctx, t)
+		if err != nil {
+			cancelFunc()
+			<-q.doneCh
+			t.Fatalf("Error starting qemu: %v", err)
+		}
+
+		beforeMem := getTotalMemory(t)
+		err = q.qmp.ExecHotplugMemory(ctx, "memory-backend-ram", "memslot1", "", 512)
```
Comment (sameo) [inline tools/CI/nats/nemu_test.go]: One nit: Could we define those `512` and `1024` as constants? We may need to reuse them for hot unplug tests...

**Comment by rbradford** [comment]
@sameo updated.

**Review by sameo — APPROVED** [review]
LGTM

**Pull request merged**